src/gui/render.cpp

```diff
@@ -1108,12 +1108,12 @@ static void init_render_settings(Section_prop& secprop)
 	        "                   (e.g., 960x720). The specified size must not be larger than\n"
 	        "                   the desktop. If it's larger than the window size, it's\n"
 	        "                   scaled to fit within the window.\n"
-	        "  N%:              Similar to 'WxH' but the size is specified as a percentage\n"
+	        "  N%%:              Similar to 'WxH' but the size is specified as a percentage\n"
 	        "                   of the desktop size.\n"
-	        "  relative H% V%:  The viewport is set to a 4:3 aspect ratio rectangle fit into\n"
+	        "  relative H%% V%%:  The viewport is set to a 4:3 aspect ratio rectangle fit into\n"
```

Comment:
Will this work with `blocksize`, `prebuffer`, and similar values, having `%s` in the config option help string?

Reply:
Oh, maybe it's more complicated than I thought.  `GetHelp()` also parses the % signs.

https://github.com/dosbox-staging/dosbox-staging/blob/3157a8c7f7604430e7e97b8715cbb22a0c84ff9f/src/misc/setup.cpp#L304-L315

My suggested change is still better than what we have.  It avoids parsing the string a 2nd time for no reason.  But it might not solve the problem if it's happening inside `GetHelp()`.

If that's the case then yeah I guess we need the double %% sign.

Reply:
Yeah we need the `%%` signs. I've added some explanatory comments to `GetHelp()`, plus I've ditched `replace_all` in favour of `format_str`. Keeping `replace_all`, though, as it's a useful string util function to have.